test/expected/license.out

```diff
@@ -0,0 +1,36 @@
+-- This file and its contents are licensed under the Apache License 2.0.
+-- Please see the included NOTICE for copyright information and
+-- LICENSE-APACHE for a copy of the license.
+\c :TEST_DBNAME :ROLE_SUPERUSER
+\set ECHO queries
+SHOW timescaledb.license;
+ timescaledb.license 
+---------------------
+ apache
+(1 row)
+
+SELECT _timescaledb_internal.tsl_loaded();
+ tsl_loaded 
+------------
+ f
+(1 row)
+
+SET timescaledb.license='apache';
+ERROR:  invalid value for parameter "timescaledb.license": "apache"
+DETAIL:  Cannot change a license in a running session.
+HINT:  Change the license in the configuration file or server command line.
+SET timescaledb.license='tsl';
+ERROR:  invalid value for parameter "timescaledb.license": "tsl"
+DETAIL:  Cannot change a license in a running session.
+HINT:  Change the license in the configuration file or server command line.
+SET timescaledb.license='something_else';
+ERROR:  invalid value for parameter "timescaledb.license": "something_else"
```

Comment:
I suggest to add a hint and print acceptable values, since it might be a typo or user tried to use Community.